docs/atomic-diff.1.md

```diff
@@ -63,6 +69,10 @@ Compare the files and RPMs (without versions) in images 'foo1' and 'foo2' and ou
 Compare only the metadata between images 'foo1' and 'foo2'
 
     atomic diff -m foo1 foo2
+    
+Compare files by 'sha256digests' and 'time' between images 'foo1' and 'foo2'
+
+    atomic diff foo1 foo2 --keywords sha256digest time
```

Comment:
FWIW using "--" between the keyword list and the image names also works, but that's not very easy to use/discover.